src/jit/target.h

```diff
@@ -1545,6 +1545,12 @@ typedef unsigned char   regNumberSmall;
 
   #define JMP_SIZE_SMALL          (4)
 
+  // The number of bytes from the end the last probed page that must also be probed, to allow for some
+  // small SP adjustments without probes. If zero, then the stack pointer can point to the last byte/word
+  // on the stack guard page, and must be touched before any further "SUB SP".
+  // For arm64, this is the maximum prolog establishment pre-indexed (that is SP pre-decrement) offset.
+  #define STACK_PROBE_BOUNDARY_THRESHOLD_BYTES 512
```

Comment:
Do we  have an assert that validates this?
"For arm64, this is the  maximum prolog establishment pre-indexed"

Reply:
No, it's "by construction". I'll look into adding this assert.